tests/pipeline/test_storage_processing_stage.py

```diff
@@ -0,0 +1,88 @@
+from copy import deepcopy
+from datetime import datetime
+
+from snuba.clickhouse.columns import ColumnSet
+from snuba.clickhouse.query import Query
+from snuba.datasets.storages.storage_key import StorageKey
+from snuba.pipeline.query_pipeline import QueryPipelineResult
+from snuba.pipeline.stages.query_processing import StorageProcessingStage
+from snuba.query import SelectedExpression
+from snuba.query.data_source.simple import Table
+from snuba.query.dsl import and_cond, binary_condition, column, equals, literal
+from snuba.query.expressions import Column, FunctionCall
+from snuba.query.query_settings import HTTPQuerySettings
+from snuba.utils.metrics.timer import Timer
+
+
+def test_basic():
+    ch_query = Query(
+        from_clause=Table(
+            "metrics_distributions_v2_local",
+            ColumnSet([]),
+            storage_key=StorageKey.METRICS_DISTRIBUTIONS,
+        ),
```

Comment:
These tests should not use existing storages/entities, they should be independent of what we already have defined